node/builder_chain.go

```diff
@@ -227,13 +226,6 @@ func ConfigFullNode(c interface{}) Option {
 
 		Override(new(dtypes.ClientBlockstore), modules.ClientBlockstore),
 
-		If(cfg.Client.UseIpfs,
```

Comment:
I believe this is the only place where `UseIpfs` is actually used, so you may as well delete that too eh?

Reply:
AH, I thought I did.